### ISSUE TITLE
Any clue on how to integrate Strider with VS online/team services?

### ISSUE THREAD

**Issue by ahmed-abdulmoniem** [Strider-CD/strider#889]
Hi Guys,

Any clue on how to integrate Strider with VS online/team services?

I have a repository on VS online (now known as VS Team Services) and I want simply to trigger the build when a code pushed to that repository?

Thanks.

**Comment by knownasilya** [comment]
If it's git based, then you could use the manual get setup, otherwise no. You'd have to write a custom provider.

**Comment by ahmed-abdulmoniem** [comment]
@knownasilya 
Yes it is a git based repository .. how I can use manual git setup?

**Comment by knownasilya** [comment]
In the Projects page there is a "Manual" tab to the right. But this won't give you auto webhooks. I'm assuming this is a service and has an API? For full Strider support, you need to write a plugin, see strider-github, strider-bitbucket or strider-gitlab.

**Comment by ahmed-abdulmoniem** [comment]
@knownasilya Can you give me a concrete example please?
